run-tests-with-build-optawebs-parent/optawebs-project-sources-test/pom.xml

```diff
@@ -70,4 +78,42 @@
             </plugin>
         </plugins>
     </build>
+    <profiles>
+        <profile>
+            <id>openshift</id>
+            <activation>
+                <property>
+                    <name>openshift</name>
+                </property>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <artifactId>exec-maven-plugin</artifactId>
+                        <groupId>org.codehaus.mojo</groupId>
+                        <executions>
+                            <execution>
+                                <id>deploy-to-openshift-and-test</id>
+                                <phase>integration-test</phase>
+                                <goals>
+                                    <goal>exec</goal>
+                                </goals>
+                                <configuration>
+                                    <executable>${project.basedir}/${project.sources.artifact}-test-script-openshift.sh</executable>
```

Comment:
`${project.basedir}` might be unreliable here, please check how this behaves when you run a single module vs from parent.

Reply:
${main.basedir} can be used if the property needs to be replaced, so you would need to define full path starting with it, probably:
`${main.basedir}/run-tests-with-build-optawebs-parent/optawebs-project-sources-test`
Though if you see no issues in current setup, disregard :-)

Reply:
Fixed